### PR TITLE
Replace typedefs in core/perf_test and core/unit_test/incremental

### PR DESCRIPTION
**Pull request by masterleinad**
Part of #3213.

### CODE REVIEW

**Review by janciesko — APPROVED** [review]
LGTM

File: core/perf_test/PerfTestMDRange.hpp [hunk 1]
```diff
@@ -216,14 +215,15 @@ struct MultiDimRangePerf3D {
...
+      //         execution_space >;
       // using tile_type = typename MDRangeType::tile_type;
       // using point_type = typename MDRangeType::point_type;
-      // Kokkos::MDRangePolicy<Kokkos::Rank<3, iterate_type::Left,
-      // iterate_type::Left>, execution_space >
-      // policy(point_type{{0,0,0}},point_type{{icount,jcount,kcount}},tile_type{{Ti,Tj,Tk}}
-      // );
+      // MDRangeType policy(point_type{{0,0,0}},
+      //                    point_type{{icount,jcount,kcount}},
+      //                    tile_type{{Ti,Tj,Tk}});
```
Comment (dalg24) [inline core/perf_test/PerfTestMDRange.hpp:226]: @janciesko @rgayatri23 Why do we have commented lines in the first place?

Comment (dalg24) [reply core/perf_test/PerfTestMDRange.hpp:226]: Please ignore I thought it was in incremental testing

**Comment by rgayatri23** [comment]
Ah ok, everything else LGTM

**Pull request merged**